src/Microsoft.AspNet.Mvc.Core/Controller.cs

```diff
@@ -1026,6 +1026,61 @@ public virtual Task<bool> TryUpdateModelAsync<TModel>([NotNull] TModel model)
                 predicate);
         }
 
+        /// <summary>
+        /// Validates the specified <paramref name="model"/> instance using values from the controller's current
+        /// <see cref="IValueProvider"/>.
+        /// </summary>
+        /// <param name="model">The model instance to update.</param>
+        /// <returns>A <see cref="bool"/> that on completion returns <c>true</c> if the <see cref="ModelState" />
+        /// is valid; <c>false</c> otherwise. </returns>
+        [NonAction]
+        public virtual bool TryValidateModel([NotNull] object model)
+        {
+            return TryValidateModel(model, prefix: null);
+        }
+
+        /// <summary>
+        /// Validates the specified <paramref name="model"/> instance using values from the controller's current
+        /// <see cref="IValueProvider"/> and a <paramref name="prefix"/>.
+        /// </summary>
+        /// <param name="model">The model instance to update.</param>
+        /// <param name="prefix">The prefix to use when looking up values in the current <see cref="IValueProvider"/>
```

Comment:
"The _key_ to use when looking up information in `<see cref="ModelState"/>`." or "... the current `<see cref="ModelStateDictionary"/>`" (prefer the first but either works)